integration_tests/models/sql/schema.yml

```diff
@@ -60,42 +60,42 @@ models:
           - dbt_utils.at_least_one
           - not_null
           - unique
-      
+
   - name: test_nullcheck_table
     columns:
       - name: field_1
         tests:
           - not_empty_string
-      
+
       - name: field_2
         tests:
           - not_empty_string
-      
+
       - name: field_3
         tests:
           - not_empty_string
-      
+
   - name: test_pivot
     tests:
       - dbt_utils.equality:
           compare_model: ref('data_pivot_expected')
-        
-  - name: test_unpivot
+
+  - name: test_unpivot_original_api
```

Comment:
I think you want to add another test for the `test_unpivot` model here, right? I can see the model code and the  expected seed file, but i don't see the corresponding test. Thanks for preserving the test for the existing behavior!

Reply:
duh, I forgot the other test.